server/src/main/java/dev/slimevr/config/OSCConfig.java

```diff
@@ -7,10 +7,10 @@ public class OSCConfig {
 	private boolean enabled = false;
 
 	// Port to receive OSC messages from
-	private int portIn;
+	private int portIn = 9002;
```

Comment:
The class gets extended and the fields get used in children, make it protected or rewrite in kotlin and make public.